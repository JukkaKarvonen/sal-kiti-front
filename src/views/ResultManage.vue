<template>
  <div class="event">
    <ResultForm />
  </div>
</template>

<script>
/**
 * Result management view
 */
import ResultForm from "@/components/ResultForm.vue";

export default {
  name: "ResultManageView",
  components: {
    ResultForm
  },
  created() {
    document.title =
      this.$t("title.result_manage") + " | " + this.$t("title.prefix");
  }
};
</script>
