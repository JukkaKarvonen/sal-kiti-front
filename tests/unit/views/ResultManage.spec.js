import { shallowMount, createLocalVue } from "@vue/test-utils";
import ResultManage from "@/views/ResultManage.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("ResultManage.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(ResultManage, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
