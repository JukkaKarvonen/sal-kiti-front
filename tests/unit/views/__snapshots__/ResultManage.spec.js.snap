// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultManage.vue renders content 1`] = `
<div
  class="event"
>
  <resultform-stub />
</div>
`;
